ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -350,8 +331,14 @@ export const AccountListMenu = ({
                     disabled={isBtcTestnetAccountAlreadyCreated}
                     size={Size.SM}
                     startIconName={IconName.Add}
-                    onClick={() => {
-                      setActionMode(ACTION_MODES.ADD_BITCOIN_TESTNET);
+                    onClick={async () => {
+                      // The account creation + renaming is handled by the Snap account bridge, so we
+                      // need to close the current model
+                      onClose();
+
+                      await createBitcoinAccount(
+                        MultichainNetworks.BITCOIN_TESTNET,
+                      );
                     }}
                     data-testid="multichain-account-menu-popover-add-account-testnet"
```

Comment:
To match with the other change you made with mainnet:

```suggestion
                    data-testid="multichain-account-menu-popover-add-btc-account-testnet"
```